gomint-server/src/main/java/io/gomint/server/network/packet/PacketTransfer.java

```diff
@@ -0,0 +1,26 @@
+package io.gomint.server.network.packet;
+
+import io.gomint.jraknet.PacketBuffer;
+import io.gomint.server.network.Protocol;
+
+public class PacketTransfer extends Packet {
+
+    public String address;
+    public int port = 19132;
+
+    public PacketTransfer() {
+        super( Protocol.PACKET_TRANSFER );
+    }
+
+    @Override
+    public void serialize( PacketBuffer buffer ) {
+        buffer.writeString( address );
+        buffer.writeLShort( (byte) port );
```

Comment:
Why cast to byte? It will overflow the byte and get negative?

Reply:
ups